### DISCUSSION TITLE
Como faço pra ganhar estrelas no GitHub?

### DISCUSSION THREAD

**Discussion by syspone** [community/community#18898] [New to GitHub] [answered]
Boa noite, 
Pessoal sou novo no GitHub, apesar que uso a plataforma mais pra postar meus projetos. Mais esses dias percebi que também possui uma função de ganhar estrelas e ser bem visto na comunidade, como faço pra realizar esse processo? o que é preciso fazer?

Obrigado.

**Comment by RonaldGuilhermePDS** [comment]
Olá!
Não acho que exista uma maneira mágica de fazer isso... mas repositórios populares são aqueles que normalmente resolvem algum problema da comunidade.

**Reply by syspone** [reply]
vlw pelo retorno amigo!

**Reply by LuanSchneider** [reply]
Ainda não conseguir entender

**Comment by jfullstackdev** [comment, marked as answer]
it's true popular projects on GitHub are really big projects that solve problems,

but a single developer can make a network on GitHub, just like what I'm doing, when I started being active across sites, sharing my projects and following users here on GitHub and Twitter, I now have more stars than ever, even if it's a simple tutorial on something, 

back then, you just need to create a unique project and it would be discovered because there were too few sites, now it's about directing the traffic so that users will see your project because it will not be seen because of the multitude of projects over the Internet

https://github.com/jdevstatic
